core/src/main/java/org/wso2/testgrid/core/command/CleanUpCommand.java

```diff
@@ -116,13 +126,21 @@ public void execute() throws CommandExecutionException {
             List<String> allTestPlans = testPlanUOW.deleteDatasourcesByAge(remainingBuildCount);
             logger.info("number of items to delete: " + allTestPlans.size());
 
+            logger.info("Clearing S3 files");
+            for (String deletingTestPlan : allTestPlans) {
+                deleteS3(deletingTestPlan);
+            }
+
+            logger.info("Deleting test plan from DB");
```

Comment:
Which test-plan? Again, missing context. Btw, I think think should be a debug log.